rust/stackable-cockpit/src/platform/demo/spec.rs

```diff
@@ -40,3 +51,76 @@ pub struct DemoSpecV2 {
     #[serde(default)]
     pub parameters: Vec<Parameter>,
 }
+
+#[derive(Debug, Snafu)]
+pub enum DemoError {
+    #[snafu(display("no stack with name '{name}'"))]
+    NoSuchStack { name: String },
+
+    #[snafu(display("stack error"))]
+    StackError { source: StackError },
+
+    #[snafu(display("cannot install demo in namespace '{requested}', only {} supported", supported.join(", ")))]
+    UnsupportedNamespace {
+        requested: String,
+        supported: Vec<String>,
+    },
+}
+
+impl DemoSpecV2 {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
This needs to be addressed. However, I'm not quite sure how. Do we wrap the arguments in a new type or do we create some kind of builder..?

Reply:
Let's leave it like this and re-visit it in the future.